### PR TITLE
Add more guidance to test review

### PR DESCRIPTION
**Pull request by p-schuler**
Small addition to the Test guidance section
Closing #108

### CODE REVIEW

File: code-reviews/process-guidance/reviewer-guidance.md [hunk 1]
```diff
@@ -89,3 +89,5 @@ Code reviews should use the below guidance and checklists to ensure positive and
 
 - Tests should always be committed in the same PR as the code itself (‘I’ll add tests next’ is not acceptable)
 - Make sure tests are sensible and valid assumptions are made.
+- Make sure edge cases are handled as well
```
Comment (ChrisRisner) [inline code-reviews/process-guidance/reviewer-guidance.md:92]: Nit: 2nd and 4th line end in periods.  First and 3rd do not.
